examples/compss/java/README.md

```diff
@@ -0,0 +1,11 @@
+# Simple in java with COMPSs and OSCAR
+
+The source code can be found in [github](https://github.com/bsc-wdc/tutorial_apps/tree/stable/python/wordcount)
+This example works by introducing a number and returning the number increased by one and module 255.
+
+In the java program, it does not need the source code. It needs a `.jar` file created with maven `mvn clean package`.
```

Comment:
Please rephrase this "In the java program, it does not need the source code." What do you mean?